src/main/java/io/opentracing/contrib/concurrent/TracedScheduledExecutorService.java

```diff
@@ -1,45 +1,83 @@
 package io.opentracing.contrib.concurrent;
 
+import io.opentracing.Scope;
 import io.opentracing.Tracer;
-
 import java.util.concurrent.Callable;
 import java.util.concurrent.ScheduledExecutorService;
 import java.util.concurrent.ScheduledFuture;
 import java.util.concurrent.TimeUnit;
 
 /**
  * @author Jose Montoya
+ *
+ * Executor which propagates span from parent thread to scheduled.
+ * Optionally it creates parent span if traceWithActiveSpanOnly = false.
  */
 public class TracedScheduledExecutorService extends TracedExecutorService implements ScheduledExecutorService {
 
-	private final ScheduledExecutorService delegate;
```

Comment:
These changes *look* OK, but I might have missed something because there are logic and formatting changes at the same time...

Reply:
I reverted this line formatting